### ISSUE TITLE
Wiki: Screenshots gone

### ISSUE THREAD

**Issue by franz-josef-kaiser** [leifeld/dna#84]
The [screenshots](https://github.com/leifeld/dna/wiki/Screenshots) – as well as other images in the wiki – are 404ing/ gone. I tried searching up the original sources in the repo as the they are linked from GitHubs own CDN `camo.githubusercontent.com`, but could not find them (and I did not want to travel back in time and go through the git logs). So no PR, still hope it helps.

**Comment by leifeld** [comment]
Oh, how weird! Sorry about that! I'll fix it over the next few days and report back. Thanks for bringing this up.

**Comment by leifeld** [comment]
OK, I have fixed the screenshots. However, the network export and time series functions are currently undergoing major revisions, so I have added the old 1.32 version screenshot for network export and removed the screenshot for time series. On the other hand, I have updated the main window screenshot to a recent beta version. I will invest more time in the WIKI and a new manual once I have worked more on the software towards a stable 2.0 release. I hope that's sufficient for now.
